### PR TITLE
Update release instructions

### PR DESCRIPTION
**Pull request by twelch**
Update the release doc after what we learned releasing 7.1.0

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -188,10 +188,13 @@ A turf release is initiated from your local computer, and then built and publish
 To make a release as a core contributor, you will need:
 - Turf repository write permission
 - Turf npm organization publish permission
-- A local copy of the Turf Github repository (not a fork!).  Starting with a fresh clone will ensure it's clean.
-  - `git clone git@github.com:Turfjs/turf.git turf-release`
+- A clean local copy of the Turf Github repository (not a fork!).  Starting with a fresh clone will ensure it's clean.
+  - Depending on your auth method
+    - `git clone git@github.com:Turfjs/turf.git turf-release`
+    - or `git clone https://github.com/Turfjs/turf.git turf-release`
```
Comment (mfedderly) [inline CONTRIBUTING.md:194]: Can you push if you do it this way?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -205,36 +208,37 @@ To make a release as a core contributor, you will need:
...
-  - If there are breaking changes, then it should be a major version bump, e.g. 7.x.x to 8.0.0.  This project follows [semantic versioning](https://semver.org/).
+- Review PR's and decide the new version number to be published. See [semantic versioning](https://semver.org/).
 
-Run the release commands, replace `7.0.0` with your version number
-- fetch the latest code from remote origin
-  - `git fetch origin`
+Run the following release commands, replacing `7.0.0` with your version number:
 
-- create a release branch, replace mf with your initials to make it clear whose branch it is.
+- create a release branch, replacing `mf` with your initials to make it clear who created the branch.
```
Comment (mfedderly) [inline CONTRIBUTING.md:215]: do you want to change `mf` to `xx` in the docs or something?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -205,36 +208,37 @@ To make a release as a core contributor, you will need:
...
+    - `git push -d origin mf/release-7.0.0`
+    - `git branch -d mf/release-7.0.0`
+  - Now redo the steps above starting with creating "A clean local copy of the Turf Github repository"
+
+- If the release action was successful, now create a Pull Request for the release to update master.
+  - You may be given a link in the output of the branch/tag push command to create the PR.
+  - If you don't get this message, just go to https://github.com/Turfjs/turf/pulls and you should be prompted at the top to create a PR for this new branch you just pushed.
+  - If that prompt doesn't appear, then just create a new pull request from the PR page and make sure the title is the version number e.g. `v7.0.0` and that it is merging your release branch -> to master.
+  - Here is an example PR - https://github.com/Turfjs/turf/pull/2615.
+  - Get approval for the release PR and merge it!
```
Comment (mfedderly) [inline CONTRIBUTING.md:240]:
```suggestion
  - If the publish action was successful, get approval for the release PR and merge it!
```

and then maybe delete the next line entirely?

File: CONTRIBUTING.md (hunk 1 above)
Comment (twelch) [reply CONTRIBUTING.md:194]: I was able to push

File: CONTRIBUTING.md (hunk 2 above)
Comment (twelch) [reply CONTRIBUTING.md:215]: yeah I can do that, make it stand out

File: CONTRIBUTING.md (hunk 3 above)
Comment (twelch) [reply CONTRIBUTING.md:240]: yep

**Review by mfedderly — APPROVED** [review]

**Pull request merged**